Makefile

```diff
@@ -31,7 +31,7 @@ else
 endif
 
 .PHONY: lint
-lint:
+lint: license
```

Comment:
Huh. Well clearly something is afoot, because builds have not been failing and we've been missing headers.

I will look into it more and consolidate the two